### DISCUSSION TITLE
Getting environment variables values

### DISCUSSION THREAD

**Discussion by jalal246** [vercel/vercel#5370] [Help] [answered]
I am trying to explore the new version of next-js integrated with Big commerce and while the whole process is fully automated I can't figure out environment variables values as they are encrypted. Is there a way to decrypt these variables?

![image](https://user-images.githubusercontent.com/19228730/98047720-1509dd00-1e35-11eb-8ada-a9339bd03dd8.png)

thanks!

**Comment by jalal246** [comment, marked as answer]
Okay. I figured it out:

- Install [Vercel CLI](https://vercel.com/download).
- Verify email to login (I am using GitHub as authentication)
- Run `vercel link` to connect with Vercel.
- Finally `vercel env pull` to pull and create `.env` locally.

**Reply by bryandandan** [reply]
Maybe you can help me with my issue. I had to open a discussion for it. After retrieving the env values, I noticed they are not being used in the repo anywhere. I tried searching for things like  `process.env.BIGCOMMERCE_STOREFRONT_API_TOKEN` but couldn't find anything. 

Any idea?

Thanks

**Reply by jalal246** [reply]
@bryandandan Pull `.env` from Vercel by doing: `vercel env pull`. This will create local `.env` matches the environment in Vercel.

**Reply by bryandandan** [reply]
As mentioned previously, I **already** pulled the variables. My question was, where in the commerce repo are they used/called?
